examples/react-nextjs/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "react-nextjs",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "media-chrome": "file:../../",
+    "next": "12.0.10",
```

Comment:
This should probably be a TypeScript app to get as much "bang for our buck" with smoke testing.

Reply:
Ahh, I was on the fence about that.  You're right though. I'll update to TS